common/men_at_arms_types/temp_draft_doc.txt

```diff
@@ -0,0 +1,604 @@
+		pikemen = 1
+		massive_infantry = 2
+		heavy_infantry = 2
+	}
+
+	buy_cost = { gold = cost_120_recruitment_cost }
+	low_maintenance_cost = { gold = cost_120_low_maint_cost }
+	high_maintenance_cost = { gold = cost_120_high_maint_cost }
+	
+	stack = 75
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = venomancer
+}
+
+webspinner = {
+	type = archers
+	
+	damage = 31
+	toughness = 24
+	pursuit = 25
+	screen = 20
+	
+	terrain_bonus = {
+		mountains = { damage = 10 toughness = 10 }
+		taiga = { damage = 10 toughness = 10 }
+	}
+
+	counters = {
+		heavy_cavalry = 1
+		light_aerial = 1
+		light_cavalry = 1
+	}
+
+	buy_cost = { gold = cost_105_recruitment_cost }
+	low_maintenance_cost = { gold = cost_105_low_maint_cost }
+	high_maintenance_cost = { gold = cost_105_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = webspinner
+}
+#Mantid
+kunchong = { 
+	type = massive_infantry
+	
+	damage = 130
+	toughness = 125
+	pursuit = 15
+	screen = 15
+	
+	terrain_bonus = {
+		steppe = { damage = 15 toughness = 15 pursuit = 15 }
+		mountains = { damage = 15 toughness = 15 pursuit = 15 }
+		plains = { damage = 10 toughness = 10 pursuit = 10 }
+	}
+
+	counters = {
+		heavy_cavalry = 1
+		skimishers = 2
+	}
+
+	buy_cost = { gold = cost_180_recruitment_cost }
+	low_maintenance_cost = { gold = cost_180_low_maint_cost }
+	high_maintenance_cost = { gold = cost_180_high_maint_cost }
+	
+	stack = 50
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = kunchong
+}
+
+#Ogre
+mauler = {
+	type = heavy_infantry
+	
+	damage = 37
+	toughness = 40
+	pursuit = 0
+	screen = 20
+
+	counters = {
+		skirmishers = 1
+		pikemen = 1
+	}
+	
+	buy_cost = { gold = cost_108_recruitment_cost }
+	low_maintenance_cost = { gold = cost_108_low_maint_cost }
+	high_maintenance_cost = { gold = cost_108_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = mauler
+}
+
+#Pandaren
+mushan_rider = {
+	type = heavy_cavalry
+	
+	damage = 185
+	toughness = 160
+	pursuit = 60
+	screen = 35
+	
+	terrain_bonus = {
+		plains = { damage = 20 toughness = 15 pursuit = 10 }
+		drylands = { damage = 20 toughness = 15 pursuit = 10 }
+		desert = { damage = 20 toughness = 15 pursuit = 10 }
+		hills = { damage = -5 }
+		jungle = { damage = 5 }
+		mountains = { damage = -10 }
+		desert_mountains = { damage = -10 }
+		wetlands = { damage = -10 }
+	}
+
+	winter_bonus = {
+		mild_winter = { damage = -4 toughness = -4 pursuit = -4 screen = -4}
+		normal_winter = { damage = -5 toughness = -5 pursuit = -5 screen = -5 }
+		harsh_winter = { damage = -6 toughness = -6 pursuit = -6 screen = -6 }
+	}
+
+	counters = {
+		archers = 2
+		caster = 1
+	}
+
+	buy_cost = { gold = cost_240_recruitment_cost }
+	low_maintenance_cost = { gold = cost_240_low_maint_cost }
+	high_maintenance_cost = { gold = cost_240_high_maint_cost }
+	
+	stack = 50
+	ai_quality = { value = culture_ai_weight_mushan }
+	icon = mushan
+}
```

Comment:
Should be regional like the Kodo, right?

Reply:
yes, is your question regarding the fact they are in the draft doc?

Reply:
The question is where they'll end up since the draft is not sorted by regional and cultural.

Reply:
Regional, I will sort once finished. I will resolve this, there is nothing to change.